content/spin/install.md

```diff
@@ -26,10 +26,14 @@ You can manage your Spin installation via [Homebrew](https://brew.sh/). Homebrew
 
 Install the Fermyon tap, which Homebrew tracks, updates, and installs Spin from:
 
+<!-- @selectiveCpy -->
+
```

Comment:
Worth changing the next line to use Markdown?  I'm thinking someone might accidentally copy the class and id to another OS tab.